README.rst

```diff
@@ -249,8 +249,8 @@ If you want a non-default data file distributed with the package just proceed as
 .. code-block:: python
 
     >>> from particle import data
-    >>> Particle.load_table(data.open_text(data, 'particle2018.csv'))
-    >>> Particle.load_table(data.open_text(data, "nuclei2020.csv"), append=True)  # I still want nuclei info
+    >>> Particle.load_table(data.files / "particle2018.csv"))
```

Comment:
Below I see some other example usage like `filename = data.files("pdgid_to_pythiaid.csv")` so wonder if both this one and the above one with a slash are viable? In any case `Particle.load_table(data.files / "particle2018.csv"))` seems to have a closing parenthesis too many.

Reply:
`filename = data.files("pdgid_to_pythiaid.csv")` Oops, that would be a typo. `data.files` is basically a Path, so it's not callable - it should be either `data.files / name` or the identical, more verbose, but more chain able `data.files.joinpath(name)`.

BTW, is "files" a good name for this? We could call it `data.basepath` or something else like that.

Reply:
Good that this was caught. Yeah, I like `data.basepath` more. Seems clearer 👍.